src/main/resources/.gitignore

```diff
@@ -0,0 +1 @@
+developer.properties
```

Comment:
This is probably no longer needed since the `developer.properties` are now included in the main `.gitignore` I think we could remove it or leave it in place as an extra precaution.